tfrecorder/beam_image.py

```diff
@@ -65,8 +65,8 @@ def load(image_uri):
   try:
     with tf.io.gfile.GFile(image_uri, 'rb') as f:
       return Image.open(f)
-  except tf.python.framework.errors_impl.NotFoundError:
-    raise OSError('File {} was not found.'.format(image_uri))
+  except tf.python.framework.errors_impl.NotFoundError as e:
+    raise OSError('File {} was not found.'.format(image_uri)) from e
```

Comment:
What does `from e` do?

Reply:
Fixes pylint mostly.  :)

How this works is that if you raised the OSError it would lose/obscure the tf.p.f.e.NotFoundError.  By explicitly saying from e, it chains the OSError to the OG error in the except clause.